README.md

```diff
@@ -54,6 +55,10 @@ might want to use a conservative version locking in your `Gemfile`:
 ```rb
 gem 'rubocop', '~> 0.52.1', require: false
 ```
+To run RuboCop using the [Docker](https://hub.docker.com/r/coder95/tiller/) image
+```
+$ docker pull coder95/rubocop
```

Comment:
I think we should host the image as `bbatsov/rubocop` or `rubocop/rubocop` or something. And it should be built with hook on this repository. We can reduce security risk with owner's or automatic build.

Reply:
@pocke created an organisation name `rubocopz` in Dockerhub. There are some blockers with creation auto-build with Dockerhub. They are
1. An organisation can have only one linked GitHub account.
2. The user who wishes to create auto-build process should have access to the repository or he should be part of the organisation.

Provide me your docker id, so that I can add to the organisation and we could proceed further.